network/controller/bootstrap/bootstrap.go

```diff
@@ -182,16 +183,33 @@ func init() {
 
 // Bootstrap on the discovery node (step 1 of the bootstrap process)
 func (bc *bootstrapper) Bootstrap(ctx context.Context) (*network.BootstrapResult, *DiscoveryNode, error) {
-	log.Info("Bootstrapping to discovery node")
+	logger := inslogger.FromContext(ctx)
+	logger.Info("Bootstrapping to discovery node")
 	ctx, span := instracer.StartSpan(ctx, "Bootstrapper.Bootstrap")
 	defer span.End()
 	ch := bc.getDiscoveryNodesChannel(ctx, bc.Certificate.GetDiscoveryNodes(), 1)
```

Comment:
The channel returned from this function has capacity of 1. Do we really get responses from all discovery nodes?